tests/css_atrule/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -308,6 +3193,12 @@ run-in
 }
 @supports not (transform-origin: 10em
 10em
+10em) {
+}
```

Comment:
Something has happened here with all the newlines and stuff.

Reply:
@lydell problem in parser, i will fix this in parser, but it is valid code